File: frontend/demo/component/notification/notification-contrast.ts
import 'Frontend/demo/init'; // hidden-source-line
import '@vaadin/button';
import { LitElement } from 'lit';
import { customElement } from 'lit/decorators.js';
import { Notification } from '@vaadin/notification';

@customElement('notification-contrast')
export class Example extends LitElement {
  firstUpdated() {
    // tag::snippet[]
    const notification = Notification.show('5 tasks deleted', {
      position: 'middle',
      duration: 0,
      theme: 'contrast',
    });
    // end::snippet[]
  }
}

File: src/main/java/com/vaadin/demo/component/notification/NotificationContrast.java
package com.vaadin.demo.component.notification;

import com.vaadin.flow.component.html.Div;
import com.vaadin.flow.component.notification.Notification;
import com.vaadin.flow.component.notification.NotificationVariant;
import com.vaadin.flow.router.Route;
import com.vaadin.demo.DemoExporter; // hidden-source-line

@Route("notification-contrast")
public class NotificationContrast extends Div {

    public NotificationContrast() {
        // tag::snippet[]
        // When creating a notification using the `show` static method,
        // the duration is 5-sec by default.
        Notification notification = Notification
                .show("5 tasks deleted");
        notification.addThemeVariants(NotificationVariant.LUMO_CONTRAST);
        // end::snippet[]
        notification.setPosition(Notification.Position.MIDDLE);
        notification.setDuration(0);
    }

    public static class Exporter extends DemoExporter<NotificationContrast> { // hidden-source-line
    } // hidden-source-line
}
